nixos/modules/config/pulseaudio.nix

```diff
@@ -178,9 +178,9 @@ in {
 
         config = mkOption {
           type = types.attrsOf types.unspecified;
-          default = {};
+          default = { flat-volumes = "no"; };
```

Comment:
If we set the default here, if the user sets the config on their own to something, this value gets overridden, which is probably not what people would expect. I think it should be set like this in the config section of this module instead:

```nix
{
  config = {
    hardware.pulseaudio.config.flat-volumes = mkDefault "no";
  };
}
```

Then people can set other values without having `flat-volumes` turned on again.

Reply:
Yes that is correct :+1: